procurement/templates/procurement/emails.html

```diff
@@ -54,8 +54,13 @@ <h1 class="h2 text-center">Get email alerts throughout the lifecycle of council
                     <label class="form-label visually-hidden" for="council_exact">Council name</label>
                     <div class="search-input">
                         <svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" viewBox="0 0 512 512" fill="currentColor" class="" role="presentation"><path d="M500.3 443.7l-119.7-119.7c27.22-40.41 40.65-90.9 33.46-144.7C401.8 87.79 326.8 13.32 235.2 1.723C99.01-15.51-15.51 99.01 1.724 235.2c11.6 91.64 86.08 166.7 177.6 178.9c53.8 7.189 104.3-6.236 144.7-33.46l119.7 119.7c15.62 15.62 40.95 15.62 56.57 0C515.9 484.7 515.9 459.3 500.3 443.7zM79.1 208c0-70.58 57.42-128 128-128s128 57.42 128 128c0 70.58-57.42 128-128 128S79.1 278.6 79.1 208z"/></svg>
-                        <input type="text" class="form-control" id="council_exact" name="council_exact">
+                        <input type="text" class="form-control{% if council_exact_error %} is-invalid {% endif %}" id="council_exact" name="council_exact">
                     </div>
+                    {% if council_exact_error %}
+                            <div class="text-danger">
```

Comment:
I think you’ll want `<div class="invalid-feedback d-block">` here, and no `<small>` tags, like I did on the homepage:

https://github.com/mysociety/contract-countdown/commit/c80045ea00e7dcd4321368d761098423d79bb5dc#diff-e16db144ae57158272954cc4636d98c85051ab3e80182a51e8f3b2c6f5da033fR11-R13

Note you need the `d-block` class because Bootstrap hides `invalid-feedback` elements unless they follow an `.is-invalid` element or an `:invalid` element inside of a `.was-validated` parent (ridiculously opinionated!) – which wasn’t the case on the homepage because of the extra wrapper div added by the autocomplete plugin, and isn’t the case here either because of the `<div class="search-input">` wrapper.

More examples of Bootstrap 5 form validation classes in the docs: https://getbootstrap.com/docs/5.2/forms/validation/

Reply:
This has now been made consistent!